CHANGELOG.md

```diff
@@ -11,6 +11,127 @@ git log --first-parent
 
 Please visit our wiki [Changelog](https://github.com/ginkgo-project/ginkgo/wiki/Changelog) for unreleased changes.
 
+## Version 1.8.0
+
+The Ginkgo team is proud to announce the new Ginkgo minor release 1.8.0. This
+release brings new features such as:
+- A brand new file-based configuration for Ginkgo objects: you can now construct
+  Ginkgo objects (solvers, preconditioners, stopping criteria, ...) from a JSON
```

Comment:
```suggestion
- A brand new file-based configuration for Ginkgo objects: you can now construct
  Ginkgo objects (solvers, preconditioners, ...) from a JSON
```
I think removing stopping criterion is better because it is only internal object not the final output